src/main.rs

```diff
@@ -22,7 +22,7 @@ fn main() {
     let release = matches.value_of("release").unwrap();
 
     let mut repository = Repository::new(matches.value_of("path").unwrap(), release);
-    let changelog = repository.changelog(matches.is_present("print-authors"));
+    let changelog = repository.changelog(matches.is_present("print-authors")).expect("Something wrong happened with gitmoji_changelog");
```

Comment:
I think you can unwrap() here, there is no error handling atm and I think if would be simplier to have the real trace than this message to debug.

WDYT?